internal/provider/resource_gitlab_project_mirror.go

```diff
@@ -83,6 +87,12 @@ import_url, mirror, and mirror_trigger_builds properties on the gitlab_project r
 				Optional:    true,
 				Default:     true,
 			},
+			"disable_on_destroy": {
+				Description: "Disables the mirror on a destroy instead of deleting it. **Note**: project mirror deletion has only available from GitLab 14.10 onwards. In v4 of the provider the default will be switched from `true` to `false`.",
```

Comment:
```suggestion
				Description: "Disables the mirror on a destroy instead of deleting it. **Note**: project mirror deletion is only available from GitLab 14.10 onwards. In v4 of the provider the default will be switched from `true` to `false`.",
```